src/parser/descriptionparser.py

```diff
@@ -0,0 +1,6 @@
+
+class DescriptionParser(object):
+
+    @classmethod
+    def parse(cls, description: str):
+        return description.split(sep="<", maxsplit=1)[0]
```

Comment:
I think that it is more convenient to extract this description.split to other method f.e left_trim_html_tags and then put a result of this inside this parse method

Reply:
Done